src/core/client.ts

```diff
@@ -216,15 +235,19 @@ export default class Client {
         class: notice.name,
         message: notice.message,
         backtrace: notice.backtrace,
-        fingerprint: notice.fingerprint
+        fingerprint: notice.fingerprint,
+        tags: notice.tags
       },
       request: {
         url: filterUrl(notice.url, this.config.filters),
         component: notice.component,
         action: notice.action,
         context: notice.context,
         cgi_data: cgiData,
-        params: filter(notice.params, this.config.filters) || {},
+        params: {
+          tags: notice.tags,
```

Comment:
We don't need to send the tags in params—just in `payload.error.tags` like you have above.